cmd/lakectl/cmd/abuse_commit.go

```diff
@@ -15,13 +15,13 @@ import (
 )
 
 var abuseCommitCmd = &cobra.Command{
-	Use:               "commit <source ref uri>",
-	Short:             "Commits to the source ref repeatedly",
+	Use:               "commit <branch uri>",
+	Short:             "Commits to the source branch repeatedly",
 	Hidden:            false,
 	Args:              cobra.ExactArgs(1),
 	ValidArgsFunction: ValidArgsRepository,
 	Run: func(cmd *cobra.Command, args []string) {
-		u := MustParseRefURI("source ref", args[0])
+		u := MustParseBranchURI("Branch", args[0])
```

Comment:
relevant to all the MustXXX changed text - we wanted it to match the value found in `Use`.
so in thicase case "branch uri" to identify the first argument.

Reply:
Done